src/stdlib/File.ts

```diff
@@ -28,6 +27,201 @@ export function getMemfsPath(fileUri: string) {
+                return BrsBoolean.False;
+            }
+        }
+    }
+);
+
+/** Deletes a file, return true if successful */
+export const DeleteFile = new Callable(
+    "DeleteFile",
+    {
+        signature: {
+            args: [
+                {name: "file", type: ValueKind.String}
+            ],
+            returns: ValueKind.Boolean
+        },
+        impl: (interpreter: Interpreter, file: BrsString) => {
+            const volume = getVolumeByPath(interpreter, file.value);
+            if (volume === null) {
+                return BrsBoolean.False;
+            }
+
+            const memfsPath = getMemfsPath(file.value);
+            try {
+                volume.unlinkSync(memfsPath);
+                return BrsBoolean.True;
+            } catch (err) {
+                return BrsBoolean.False;
+            }
+        }
+    }
+);
+
+/** Deletes a directory (if empty), return true if successful */
+export const DeleteDirectory = new Callable(
+    "DeleteDirectory",
+    {
+        signature: {
+            args: [
+                {name: "dir", type: ValueKind.String}
+            ],
+            returns: ValueKind.Boolean
+        },
+        impl: (interpreter: Interpreter, dir: BrsString) => {
+            const volume = getVolumeByPath(interpreter, dir.value);
+            if (volume === null) {
+                return BrsBoolean.False;
+            }
+
+            const memfsPath = getMemfsPath(dir.value);
+            try {
+                volume.rmdirSync(memfsPath);
+                return BrsBoolean.True;
+            } catch (err) {
+                return BrsBoolean.False;
+            }
+        }
+    }
+);
+
+/** Creates a directory, return true if successful */
+export const CreateDirectory = new Callable(
+    "CreateDirectory",
+    {
+        signature: {
+            args: [
+                {name: "dir", type: ValueKind.String}
+            ],
+            returns: ValueKind.Boolean
+        },
+        impl: (interpreter: Interpreter, dir: BrsString) => {
+            const volume = getVolumeByPath(interpreter, dir.value);
+            if (volume === null) {
+                return BrsBoolean.False;
+            }
+
+            const memfsPath = getMemfsPath(dir.value);
+            try {
+                volume.mkdirSync(memfsPath);
+                return BrsBoolean.True;
+            } catch (err) {
+                return BrsBoolean.False;
+            }
+        }
+    }
+);
+
+/** Stubbed function for formatting a drive; always returns false */
+export const FormatDrive = new Callable(
+    "FormatDrive",
+    {
+        signature: {
+            args: [
+                {name: "drive", type: ValueKind.String},
+                {name: "fs_type", type: ValueKind.String}
+            ],
+            returns: ValueKind.Boolean
+        },
+        impl: (interpreter: Interpreter, dir: BrsString) => {
+            console.warn("`FormatDrive` is not implemented in `brs`.");
```

Comment:
This'll warn every time `formatDrive()` is called, right?  That's probably better than warning once like we do for `RebootSystem`.

Reply:
Right.  The thing I don't like about this is there's a warning now whenever we run the tests, which is an eye-sore and will cause confusion later on.  I was wondering if we should wrap this in some env check to only warn in release builds.

Reply:
Good idea!  We could definitely do that – similar things already happen in [Error.ts](https://github.com/sjbarag/brs/blob/0719932c49e9e7fe9e709f97b91966320dc42921/src/Error.ts#L13-L15).  Also try replacing this with `console.error`, because the `stderr` output tends to get silenced for unit tests anyway (at least I thought?)

Reply:
I see console.warn in the output.  According to the docs _warn_ is an alias for _error_, so it also goes to _stderr_.  

https://nodejs.org/api/console.html#console_console_warn_data_args